core/templates/dev/head/pages/exploration-editor-page/modal-templates/post-publish-modal.template.html

```diff
@@ -47,7 +47,12 @@
     display: inline-block;
     padding: 10px;
     margin-bottom: 2px;
+    margin-top: 10px;
```

Comment:
Removing padding will help here? (not sure, I'm fine with this as well. Asking as lots of random *px are used in this CSS.)

Reply:
No, padding is inside the text field, between text and the border, it would look ugly.

Reply:
Thanks for the explanation. (Marking this resolved!)